src/Particle/tests/test_particle.cpp

```diff
@@ -27,6 +27,44 @@ using std::string;
 
 namespace qmcplusplus
 {
+
+class TestMinimalTraitCPU {};
+
+template<>
+struct ParticleSet_minimal_trait<TestMinimalTraitCPU>
+{
+  using tag = ParticleSet::MinimalCPU;
+};
+
+TEST_CASE("ParticleSet minimal copy constructors", "[particle]")
+{
+  Communicate* comm;
+  comm = OHMMS::Controller;
+  outputManager.pause();
+
+  MinimalParticlePool mpp;
+  ParticleSetPool particle_pool = mpp(comm);
+  auto& pset_target = *(particle_pool.getParticleSet("e"));
+  auto& pset_source = *(particle_pool.getParticleSet("ion"));
+  int table_id = pset_target.addTable(pset_source, DTModes::ALL_OFF);
+  ParticleSet pset_minimal_default(pset_target, ParticleSet::MinimalDefault{});
+  // In the cass of ENABLE_OFFLOAD the whole DistanceTable creation ignores DTModes::ALL_OFF which
+  // isn't really a flag.
```

Comment:
It is not ignoring DTModes::ALL_OFF but means optional behaviors are not requested from this add.

Reply:
I'll remove my whining, I think I added enough docs elsewhere that it's no longer a surprise.